cdap-data-fabric/src/test/java/co/cask/cdap/data2/util/hbase/HBaseVersionTest.java

```diff
@@ -74,6 +74,14 @@ public void testHBaseVersionToCompatMapping() throws ParseException {
     assertCompatModuleMapping(HBaseVersion.Version.HBASE_11, "1.2.0-IBM-7");
     assertCompatModuleMapping(HBaseVersion.Version.UNKNOWN, "1.3.0");
 
```

Comment:
We don't support CDH 4.7.1 [source](https://docs.cask.co/cdap/4.2.0/en/admin-manual/cdap-hadoop-compatibility.html).
Will add the bigtop one though.